AUTHORS.md

```diff
@@ -0,0 +1,2 @@
+Stegner, Wayne <stegnerw>
```

Comment:
Make these items. Also it does not display our tags. Make them proper links to our GitHub URLs.